examples/simple/simple.tf

```diff
@@ -46,6 +46,7 @@ module "fargate_api" {
   vpc_id                        = module.acs.vpc.id
   codedeploy_service_role_arn   = module.acs.power_builder_role.arn
   role_permissions_boundary_arn = module.acs.role_permissions_boundary.arn
+  xray_enabled                  = true
```

Comment:
(nit) I've tried to keep the simple example as bare bones as possible so I would remove this since it has a default, but this is fine too since it's just a boolean flag. No strong opinion either way.

Reply:
I will change that.